php-frontend/src/main/java/org/sonar/php/tree/impl/expression/FunctionCallTreeImpl.java

```diff
@@ -37,24 +37,24 @@
   private static final Kind KIND = Kind.FUNCTION_CALL;
   private ExpressionTree callee;
   private final InternalSyntaxToken openParenthesisToken;
-  private final SeparatedListImpl arguments;
+  private final SeparatedListImpl<ExpressionTree> arguments;
   private final InternalSyntaxToken closeParenthesisToken;
 
-  public FunctionCallTreeImpl(ExpressionTree callee, InternalSyntaxToken openParenthesisToken, SeparatedListImpl arguments, InternalSyntaxToken closeParenthesisToken) {
+  public FunctionCallTreeImpl(ExpressionTree callee, InternalSyntaxToken openParenthesisToken, SeparatedListImpl<ExpressionTree> arguments, InternalSyntaxToken closeParenthesisToken) {
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Split this 184 characters long line (which is greater than 180 authorized). [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://dory.sonarsource.com/coding_rules#rule_key=squid%3AS00103)